docs/examples/map_vs_mean.md

```diff
@@ -0,0 +1,150 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernel_info:
+  name: psignifit
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Parameters estimates: MAP and mean
+
+In this demo, we build a data set with a wide posterior, and show how to select the estimate type.
+
+In `psignifit`, there are two estimate type:
+- with `MAP` (Maximum A Posteriori), the parameters are estimated as the mode of the posterior. This is the default estimate type.
+- with `mean`, the parameters are estimated as the mean of the posterior
+
+
+```{code-cell} ipython3
+import warnings
+
+from matplotlib import pyplot as plt
+import numpy as np
+
+import psignifit
+import psignifit.psigniplot as psp
+from psignifit import sigmoids
+```
+
+We generate data from a mixture of three sigmoids with different width, with
+only a few trials. The posterior over width is broad and asymmetrical, so the
+MAP and mean estimate are not identical.
+
+```{code-cell} ipython3
+# Parameters of the sigmoids and data generation
+stim_range = [0.001, 0.001 + 10 * 1.1]
+lambda_ = 0.0232
+gamma = 0.1
+
+widths = [1.1, 3.1, 6.3]
+threshold = stim_range[1] / 2
+
+sigmoid = sigmoids.Gaussian()
+num_trials = 3
+nsteps = 20
+stimulus_level = np.linspace(stim_range[0], stim_range[1], nsteps)
+
+```
+Using the `tools.psychometric` we can simulate percent correct values for each stimulus level
+
+```{code-cell} ipython3
+perccorr1 = sigmoid(stimulus_level, threshold, widths[0], gamma, lambda_)
+perccorr2 = sigmoid(stimulus_level, threshold, widths[1], gamma, lambda_)
+perccorr3 = sigmoid(stimulus_level, threshold, widths[2], gamma, lambda_)
+perccorr = np.concatenate((perccorr1, perccorr2, perccorr3))
+levels = np.concatenate((stimulus_level, stimulus_level, stimulus_level))
+```
+
+
+We construct our data array
+
+```{code-cell} ipython3
+random_state = np.random.RandomState(883)
+
+ntrials = np.ones(nsteps * 3, dtype=int) * num_trials
+hits = random_state.binomial(ntrials, perccorr)
+
+levels = np.concatenate((stimulus_level, stimulus_level, stimulus_level))
+data = np.dstack([levels, hits, ntrials]).squeeze()
+
+```
+
+This is what the data looks like -- as you can see, the data could be compatible with a wide range of psychometric functions. This means that the posterior distribution over parameters is going to be wide.
+
+```{code-cell} ipython3
+fig, ax = plt.subplots()
+ax.scatter(levels, hits)
+ax.set_xlabel("Stimulus Level")
+ax.set_ylabel("Hits")
```

Comment:
please add a `;` at the end of the line, to avoid a Text object being displayed

<img width="329" alt="image" src="https://github.com/user-attachments/assets/c0986ed3-1177-4445-bf50-0d41bbede8a3" />